pkg/systemd/services/service-details.jsx

```diff
@@ -529,7 +533,25 @@ export class ServiceDetails extends React.Component {
 
         const tooltipMessage = enabled ? _("Stop and disable") : _("Start and enable");
         const hasLoadError = this.props.unit.LoadState !== "loaded" && this.props.unit.LoadState !== "masked";
-        const loadError = this.props.unit.LoadError ? this.props.unit.LoadError[1] : null;
+
+        if (hasLoadError) {
+            const path = "/system/services" + (this.props.owner === "user" ? "#/?owner=user" : "");
+            const loadError = this.props.unit.LoadError ? this.props.unit.LoadError[1] : null;
+            const title = loadError || _("Failed to load unit");
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18422-20230303-165501-04bdc094-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)